content/reference/plugins/azure_devops/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Azure DevOps Pipelines"
+description: "Learn how to use the CI script with Azure DevOps Pipelines to run your simulations."
+lead: "Run your Gatling Enterprise simulations from Azure DevOps Pipelines."
+date: 2023-11-09T15:25:27:+0000
+lastmod: 2023-11-09T15:25:27:+0000
+weight: 23035
+---
+
+We do not currently provide an official Azure Pipelines extension.
+This page documents how to use our [CI script]({{< ref "../script" >}}) to run tests on Gatling Enterprise with Azure DevOps
+
+{{< alert info >}}
+All the configuration shown in this page is available on the demo project [gatling/azure-devops-demo](https://github.com/gatling/azure-devops-demo).
+{{< /alert >}}
+
+## Requirements
+
+You will need:
+
+- A git repository: we will be using GitHub in our example
+- A working simulation on [Gatling Enterprise Cloud](https://cloud.gatling.io): you will need its ID later on, which you can copy with **Copy Simulation ID to clipboard** in the Simulations page
+- An [API Token]({{< ref "../../admin/api_tokens" >}}) with the permissions to configure the previous simulation
+
+On Azure DevOps, your will need:
+
+- An Azure DevOps project
+- An agent pool:
+  - You can learn how to create one by following the official [Azure Devops documention](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents)
+  - You will need the agent pool's name later on, in our example we will use `AzurePipelinesAgentsPool`
+
+## Preparing the repository
+
+First, [download the CI script](https://downloads.gatling.io/releases/frontline-ci-script/{{< var ciPluginsVersion >}}/frontline-ci-script-{{< var ciPluginsVersion >}}.zip), unzip it and put the content at the root of the repository. Make sure it is named `start_simulation.sh`.
+
+{{< alert info >}}
+If you want to learn more about this CI script, you can check its [documentation page]({{< ref "../script/#using-a-shell-script" >}}).
+{{< /alert >}}
+
+Create a file called `azure-pipelines.yml` at the root of the repository with the following content:
+
+```yaml
+trigger: none
+
+pool: AzurePipelinesPool # Name of the agent pool configured in Azure DevOps
+
+steps:
+- task: Bash@3
+  displayName: Install JQ
+  inputs:
+    targetType: 'inline'
+    # Curl and JQ are required by the CI script
+    script: 'sudo apt-get update && sudo apt-get -y install jq'
```

Comment:
The comment mention both curl and jq, but the script only install jq.

Reply:
Add "In this example, we use an Ubuntu image which comes with curl but not jq"